components/Hero/index.js

```diff
@@ -41,17 +41,31 @@ class Hero extends Component {
 
     var backgroundColor = nameToRgba(color) || nameToRgba(Hero.defaultProps.color);
     var contentStyle = childrenPosition === 'after' ? {bottom: 'auto', top: '10em'} : EMPTY_OBJECT;
+    var innerCap = title;
+
+    if(this.props.title == 'Skookum'){
+      innerCap = React.DOM.img({src:'/public/images/wordmark.svg', alt:title});
+    }
+
+    if(this.props.title == 'Contact Us' || this.props.title == 'Innovation Camp'){
+      style.height = 90;
```

Comment:
this seems like a really dirty approach to this (granted, I have more dirty things in Hero than probably any other component already). :| The `Hero` should be pretty agnostic to what is coming in. Which issue is this trying to solve? I’d like to think of a cleaner way to manage it.